nservicebus/upgrades/perfcounters-3to4.md

```diff
@@ -30,6 +30,16 @@ Along with the MsBuild package deprecation, the `PerformanceCounterSettingsAttri
 
 The `ScriptPromotionPath` property is no longer supported. The new package will no longer promote the script(s) outside of the build directory. If this is required, a [post-build event](https://docs.microsoft.com/en-us/cpp/build/how-to-use-build-events-in-msbuild-projects) can be used to copy the output outside the build directory.
 
+## Counter descriptions changed
```

Comment:
Does this need to be mentioned elsewhere in the documentation and not just the upgrade guide?

Reply:
No, it's hardly even a thing.